includes/utils.php

```diff
@@ -302,47 +316,11 @@ function distributable_post_types() {
 	 * @hook distributable_post_types
 	 *
 	 * @param {array} Post types that are distributable. Default all post types except `dt_ext_connection` and `dt_subscription`.
+	 * @param {string} The type of output to return. Accepts post type 'names' or 'objects'. Default 'names'.
```

Comment:
This is a back-compat break for developers using this filter: they may get objects instead of names without any warning.

Instead, I would leave the filter unchanged (ie, always passing names) and if the function is called with `$output !== 'names'` then convert the post types to objects once the filter is run.

This very thing hit me while branch testing as I had a filter set up to `return [ 'post' => 'post' ];` so the list was throwing PHP errors.

```suggestion
```